src/main/java/core/basesyntax/service/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private final String comma = ",";
```

Comment:
Let’s  use constant variable 
```suggestion
    private static final String COMA_SEPARATOR = ",";
```